src/i18n/en-US.json

```
@@ -501,6 +501,8 @@
   "conversationJoin.namePlaceholder": "Your name",
   "conversationJoin.noAccountHead": "Don't have an account?",
   "conversationJoin.subhead": "Join conversation as temporary guest (access expires after 24 hours)",
+  "conversationJoin.termsAcceptanceText": "I accept",
+  "conversationJoin.termsLink": "Wire’s Terms of Use",
```

Maybe we could use a variable instead of Wire?
Someting like this
```suggestion
  "conversationJoin.termsLink": "{Brandname}’s Terms of Use",
```